quickstarts/video/video-web/video-web-akamai/config.yml

```diff
@@ -12,9 +12,15 @@ keywords:
   - video
   - tracking
   - Akamai
+
 documentation:
   - name: Video Akamai Installation Docs
     url: https://github.com/newrelic/video-akamai-js
```

Comment:
```suggestion
    url: >-
       https://github.com/newrelic/video-akamai-js
```

Reply:
I'd also recommend a description here so you can populate that in the documentation tile https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#documentation